main/getting-started/start-a-project.md

```diff
@@ -35,70 +32,36 @@ This presentation includes starting a project, but note an outdated detail:
 Use the [Agoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
 and put it in a `demo` directory _not located in your `agoric-sdk` clone_:
 
-
 ```sh
 # Terminal 1
-# Don't use your agoric-sdk clone as the parent of the demo directory.
-# It doesn't have to be your home directory; any other directory will do.
```

Comment:
why drop this line?

Reply:
it's still there just consolidated and bumped up to line 37 - also i do not believe we need two more lines inside of the code block for it immediately after mentioning it above the code block as well...